autoscheduler/frontend/src/App.tsx

```diff
@@ -1,9 +1,11 @@
-import * as React from "react"
+import * as React from 'react';
 
-export class App extends React.Component {
-    render() {
-        return <div>
-            Oi, put stuff here
-        </div>
-    }
-}
\ No newline at end of file
+const App: React.SFC = function App() {
+  return (
+    <div>
+      Oi, put stuff here
+    </div>
+  );
+};
+
+export default App;
```

Comment:
Also, on the React tutorial this is how they suggest making the classes

Reply:
ESLint for React recommends that stateless components be written as functional components rather than class-based components. If we expect that our top-level App component will never manage any state, then we should leave it as a functional component. If, on the other hand, we expect that in the future, the top-level App will be managing some aspects of state, then I will go ahead and throw in a dummy state to make the linter happy until we decide exactly what that state should be.

Reply:
> If we expect that our top-level App component will never manage any state, then we should leave it as a functional component.

I think this should be safe to assume but I'm not positive. As you have more front-end experience than both of us, I'll leave it up to you.

Reply:
Since we will be using Redux to manage application-wide state, I have left App.tsx as a functional component.

Reply:
Oh yeah good call, resolving conversation